src/__tests__/operate.test.js

```diff
@@ -0,0 +1,17 @@
+import operate from '../logic/operate';
+
+test('calculate the sum of 2 numbers', () => {
+  expect(operate(3, 4, '+')).toBe('7');
+});
+
+test('calculate the deduction of 2 numbers', () => {
+  expect(operate(10, 4, '-')).toBe('6');
+});
+
+test('calculate the deduction of 2 numbers', () => {
+  expect(operate(10, 4, '-')).toBe('6');
+});
+
+test('calculate the deduction of 2 numbers', () => {
+  expect(operate(2, 4, 'x')).toBe('8');
```

Comment:
- [ ] Please add tests for other operators: `+`, `%` and also test for edge cases like division by zero.

Reply:
Ok